kroxylicious-operator/packaging/install/03.Deployment.kroxylicious-operator.yaml

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+# The Deployment for the operator
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kroxylicious-operator
+  namespace: kroxylicious-operator
+  labels:
+    app.kubernetes.io/name: kroxylicious
+    app.kubernetes.io/component: operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kroxylicious
+  template:
+    metadata:
+      labels:
+        app: kroxylicious
+    spec:
+      serviceAccountName: kroxylicious-operator
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      containers:
+        - name: operator
+          image: kroxylicious.io/operator:$[container.tag]
```

Comment:
This is the only thing of interest in the `packaging` directory (which is otherwise a straight copy of kroxylicious-operator/install and kroxylicious-operator/examples). The `$[]` gets replaced by maven resource filtering.